raiden-cli/src/routes/api.v1.ts

```diff
@@ -17,12 +17,18 @@ import { makeTokensRouter } from './tokens';
 export function makeApiV1Router(this: Cli): Router {
   const router = Router();
 
-  router.use('/channels', makeChannelsRouter.call(this));
-  router.use('/tokens', makeTokensRouter.call(this));
-  router.use('/pending_transfers', makePendingTransfersRouter.call(this));
-  router.use('/connections', makeConnectionsRouter.call(this));
-  router.use('/payments', makePaymentsRouter.call(this));
-  router.use('/_testing', makeTestingRouter.call(this));
+  let status = 'syncing';
```

Comment:
Did you investigate this behavior compared to the python node? I'm also OK with pushing this into a follow-up PR in order to get this merged soon.

Reply:
Unfortunately not, let's leave this to a follow up PR, there's CLI integration of the `suggestPartners` functionality, maybe we can do both in the same PR, since this one would be just sanitization